src/components/tooltip/theme.css

```diff
@@ -14,6 +14,10 @@
   margin-top: var(--tooltip-space);
 }
 
+.hx-dropdown.hx-dropdown-up{
+    margin-top: calc(-1 * var(--tooltip-space));
```

Comment:
Could you also remove the `margin-bottom` rule(s) form the `.hx-tooltip` and `.hx-tooltip.hx-dropdown-down` as they aren't needed with this change and can cause more overlapping issues 😓

Reply:
Yes, I checked out and it appears they're good to get rid off.  Adding the change